tests/ut/host-check/job.bats

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bats
+
+load "../utils.bash"
+
+export TEMPLATE_UNDER_TEST="templates/job.yaml"
+
+setup_file () {
+    cd "$(host-check_chart_dir)" || exit
+    helm dependency update .
+}
+
+@test "host-check Job: Name consists of the release name and chart name" {
+    template --set 'targetPlatforms[0]=xrd-vrouter'
```

Comment:
It's a shame that the targetPlatforms has to be set for each success test case.

I wonder if it would be better to move `template` into a per-chart `./utils.bash`.  This can use a `template_no_set` from `../utils.bash`, which templates without setting, checks success, yamllint, kubeconform, etc.

Each per-chart MUT loads `./utils.bash` instead.

The `chart_dir` functions can then be moved into the per-chart `./utils.bash`s.